src/Psalm/Internal/Analyzer/Statements/Expression/Call/FunctionCallAnalyzer.php

```diff
@@ -562,15 +579,6 @@ private static function handleNamedFunction(
             }
 
             if ($codebase->functions->params_provider->has($function_call_info->function_id)) {
-                ArgumentsAnalyzer::analyze(
```

Comment:
Are args already analysed at this point?

Reply:
No, not yet .
I introduced a pre-analysis step here so that the type info of the parameters would be available for this plugin.

It was a bad idea. More context, see
https://github.com/vimeo/psalm/issues/7450